src/DssDirectDepositAaveDai.sol

```diff
@@ -84,15 +54,6 @@ interface InterestRateStrategyLike {
     function getMaxVariableBorrowRate() external view returns (uint256);
 }
 
-interface RewardsClaimerLike {
-    function claimRewards(address[] calldata assets, uint256 amount, address to) external returns (uint256);
-}
-
-interface EndLike {
-    function debt() external view returns (uint256);
-    function skim(bytes32, address) external;
-}
-
 contract DssDirectDepositAaveDai {
```

Comment:
For the integration functions, it might be worthwhile to provide an abstract contract that these integrations can inherit from that include the required unimplemented abstract functions. These functions are going to need to be heavily documented so that external developers know exactly what a "normalized" value should represent and this abstract contract may be a good place to store that documentation as well.

Reply:
Yes agree with an interface or abstract contract to extend for compliance. [EIP-4626](https://eips.ethereum.org/EIPS/eip-4626) has standardized language around shares vs underlying asset which would correspond to DAI vs aDAI and DAI vs cDAI. I suggest we copy the language.